File: app/src/main/kotlin/com/htoyama/likit/ui/home/liked/HomeLikedFragment.java
package com.htoyama.likit.ui.home.liked;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.htoyama.likit.R;

/**
 *
 */
public class HomeLikedFragment extends Fragment {

  @Override
  public View onCreateView(LayoutInflater inflater, ViewGroup container,
                           Bundle savedInstanceState) {
    return inflater.inflate(R.layout.fragment_home_liked, container, false);
  }

}

File: app/src/main/kotlin/com/htoyama/likit/ui/home/liked/HomeLikedFragment.kt
package com.htoyama.likit.ui.home.liked

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.htoyama.likit.R

/**
 *
 */
class HomeLikedFragment : Fragment() {

  companion object {
    fun new(): HomeLikedFragment = HomeLikedFragment()
  }

  override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                            savedInstanceState: Bundle?): View? {
    return inflater!!.inflate(R.layout.fragment_home_liked, container, false)
  }

}
